Chapter1/index.rst

```diff
@@ -19,6 +19,7 @@ A pixel can be classified by its state, which might change over time; for exampl
 State transitions can be due to fire, harvest, and succession disturbances. These transitions impact the stocks and flows of carbon pools.
 
 
```

Comment:
Can it be:
```
** Simulation models, States, and Transition**
```

**Question as a reader**: "What's a simulation model? || What landscapes?"

**Suggestion**: Do you think we can add two images of an example landscape? Both images years apart, and then dividing the images into squares to represent the pixels?

Reply:
where would i get the images from ?

Reply:
You can either sketch your own, ask for help from the community, or use publicly available pictures (e.g. from research papers) with attribution the original source.